test/data/incorrect_headers_manifest.csv

```diff
@@ -1,2 +1,2 @@
-Well Positio,sciname,Dono,ender,supname,Phenotyp,Tissue,cancer
-A:1,homo sapiens,some guy,female,abc,123,lysed cells,tumour
+Well Positio,sciname,Dono,ender,supname,Phenotyp,Tissue,cancer,taxon_id
```

Comment:
It'd be better if the header name was incorrect so it won't be matched by the regex, but this will also require adjusting the test for incorrect_headers_manifest as that now expects this row to be matched by default.